src/test/java/org/folio/circulationbff/service/UserServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.circulationbff.service;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.folio.circulationbff.domain.dto.User;
+import org.folio.circulationbff.domain.dto.UserCollection;
+import org.folio.circulationbff.service.impl.UsersServiceImpl;
+import org.folio.spring.service.SystemUserScopedExecutionService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class UserServiceTest {
```

Comment:
Apart from unit tests, there should be an API test (similar to `RequestsApiTest` or `SearchInstancesApiTest`) that would ensure that we're actually calling the endpoint of `mod-users` **of the correct tenant** using the **correct query**.

Reply:
done